src/c++/perf_analyzer/schema.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer/output",
+    "title": "Perf Analyzer output data",
+    "description": "A json file describing the output from a perf analyzer run",
+    "type": "object",
+    "properties": {
+        "experiments": {
+            "description": "The array of all experiments run by perf analyzer",
+            "type": "array",
+            "items": {
+                "type": "object"
+            },
+            "minItems": 1,
+            "uniqueItems": true,
+            "properties": {
+                "experiment": {
+                    "description": "A single experiment run by perf analyzer",
+                    "type": "object",
+                    "minItems": 1,
+                    "maxItems": 1,
+                    "properties": {
+                        "mode": {
+                            "description": "Operating mode of perf analyzer: either concurrency or request rate",
```

Comment:
This feels limiting. Can it be "for example, concurrency or request rate"

Reply:
> Do we have anything to describe the strict sequences mode or whatever?

On this thread, I think we have to draw the line somewhere on what configuration options we include in this profile export file. Since the experiments only sweep over concurrency and request rate, I think those are a suitable minimum. Of course each experiment also changes depending on many other options to PA. I think let's just leave others out unless they're asked for in the future.

Reply:
I also understood the "strict sequences" mode to be a subset of request rate mode. I am happy to update the mode options to include "strict sequences" but that will be a harder thing to determine early on. I would prefer to get this file to MVP status for Piotr and have that be a feature for the future. Basically rephrasing what Matt said.

Reply:
These are just strings and not an enum like you mentioned, so this is all fine and allows for future changes without breaking the past.